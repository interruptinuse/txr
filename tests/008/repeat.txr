@(bind a ("a" "b" "c"))
@(output)
@(repeat :counter (i 1) :vars ((x (len a))))
@i @x @a
@(end)
@(end)
