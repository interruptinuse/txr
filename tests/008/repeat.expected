1 3 a
2 3 b
3 3 c
